### PR TITLE
refactored ArrayUtils to reuse the getLength(Object array) method

### PR DESCRIPTION
**Pull request by orionlibs**
refactored ArrayUtils to reuse the getLength(Object array) method inside the class instead of calling Array.getLength()

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/ArrayUtils.java [hunk 1]
```diff
@@ -655,7 +655,7 @@ private static Object add(final Object array, final int index, final Object elem
             Array.set(joinedArray, 0, element);
             return joinedArray;
         }
-        final int length = Array.getLength(array);
+        final int length = getLength(array);
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/lang3/ArrayUtils.java:658]: -1: Why? `array` can't be null here.

File: src/main/java/org/apache/commons/lang3/ArrayUtils.java [hunk 2]
```diff
@@ -1668,7 +1668,7 @@ public static boolean containsAny(final Object[] array, final Object... objectsT
      */
     private static Object copyArrayGrow1(final Object array, final Class<?> newArrayComponentType) {
         if (array != null) {
-            final int arrayLength = Array.getLength(array);
+            final int arrayLength = getLength(array);
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/lang3/ArrayUtils.java:1671]: -1: Makes no sense since you're in a block where `array` can't be null.

**Review by garydgregory — CHANGES_REQUESTED** [review]
I think this PR should be closed, see comments.

File: src/main/java/org/apache/commons/lang3/ArrayUtils.java (hunk 1 above)
Comment (orionlibs) [reply src/main/java/org/apache/commons/lang3/ArrayUtils.java:658]: just for code reuse. This class already has a getLength method. Why not reuse it inside this class? That was my thought. Not nulls. But it is OK. I will close it

**Pull request closed without merging**